azure-pipelines.release-fluentui.yml

```diff
@@ -1,74 +1,195 @@
+    value: true
+  - name: nightlyRelease # used by nightly built docsite to setup codesandbox correctly
+    value: true
+  - name: fluentuiTarballs # temporary directory name under workspace root, used to store all fluent ui nightly built tarballs
+    value: fluentui-nightly
+
+  # below are variables configurable at queue time, so they cannot appear in yml file
+  # - name: publishDocsiteOnly # set it to true to only run Job_build_publish_doc
+  #   value: false
+  # - name: publishOfficial # set it to true to publish package to npm, and publish docsite for official releases
+  #   value: false
+  # - name: version # version to publish to npm; only make sense when publishOfficial=true
+  #   value: canary # value can only be canary/patch/minor
 
 pool: 'Self Host Ubuntu'
 
-steps:
-  - task: NodeTool@0
-    inputs:
-      versionSpec: '10.x'
-    displayName: 'Install Node.js'
-
-  - script: |
-      ver=`node -p "require('./packages/fluentui/react-northstar/package.json').version"`
-      echo "##vso[task.setvariable variable=deployBasePath]$ver"
-      echo "##vso[task.setvariable variable=docsiteVersion]$ver"
-      echo "Docsite base path published to for version $ver"
-    displayName: Set base path and version variables for job
-  - script: |
-      git config user.name "Fluent UI Build"
-      git config user.email "fluentui-internal@service.microsoft.com"
-      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
-    displayName: Authenticate git for pushes
-  - task: Bash@3
-    inputs:
-      filePath: yarn-ci.sh
-    displayName: yarn
-  - script: |
-      PRODUCTION=true yarn build
-    displayName: yarn build
-  - script: |
-      NODE_ENV=production yarn build:fluentui:docs
-    displayName: yarn build:fluentui:docs
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish docsite as pipeline artifact'
-    inputs:
-      PathtoPublish: 'packages/fluentui/docs/dist'
-      ArtifactName: 'docsite_v$(docsiteVersion)'
-
-  - task: AzureUpload@2
-    displayName: Upload docs site
-    inputs:
-      SourcePath: 'packages/fluentui/docs/dist'
-      azureSubscription: 'UI Fabric (private)'
-      storage: fluentsite
-      ContainerName: '$web'
-      BlobPrefix: '$(deployBasePath)'
-      # uncomment to deploy to prod
-      # storage: fluentsite
-      # BlobPrefix: ''
-# No NPM release in CI for now
-#  - script: |
-#      yarn test
-#    displayName: yarn test
-#
-#  - script: |
-#      yarn lint
-#    displayName: yarn lint
-#
-#  - script: |
-#      yarn release:fluentui:$(bumpType)
-#    displayName: yarn release:fluentui:$(bumpType)
+jobs:
+  - job: Job_build_publish
+    workspace:
+      clean: all
+    displayName: Build and Release Fluent Packages
+    # skip this job if publishDocsiteOnly is true
+    condition: and(succeeded(), eq(variables.publishDocsiteOnly, false))
+
+    steps:
+      - template: .devops/templates/tools.yml
+
+      # TODO remove it status
```

Comment:
Not sure what it means... Is it actionable item or cruft?

Reply:
Oh cruft